docs/5.0/preview/teleport-application-access.md

```diff
@@ -0,0 +1,208 @@
+|-|-|
+| `auth_servers` | Address of Auth or Proxy Service setup above |
+| `auth_token` | Token used to connect to other Teleport processes  |
+
+```yaml
+teleport:
+  # nodename of machine running process.
+  nodename: wellington
+  # The Auth token `tctl tokens add --type=app` or the static token. 4c7e15
+  auth_token: "4c7e15"
+  # This is the location of the Teleport Auth Server or Public Proxy
+  auth_servers:
+    - teleport.asteroid.earth:3080
+auth_service:
+  enabled: no
+proxy_service:
+  enabled: no
+ssh_service:
+   enabled: no
+# The app_service is new
+app_service:
+   enabled: yes
+   apps:
+   - name: "jwt"
+     # URI and Port of Application. If Teleport is installed
+     uri: "http://10.0.1.27:8000"
+     # This version requires a public_addr for all Apps, these
+     #  applications should have a certificate and DNS setup
+     public_addr: "jwt.asteroid.earth"
+     # Optional Labels
+     labels:
+        name: "jwt"
+     # Optional Dynamic Labels
+     commands:
+     - name: "os"
+       command: ["/usr/bin/uname"]
+       period: "5s"
+   - name: "arris"
+     uri: "http://localhost:3001"
+     public_addr: "arris.asteroid.earth"
+   # Teleport Application Access can be used to proxy any HTTP Endpoint
+   # Note: Name can't include any spaces
+   - name: "hackernews"
+     uri: "https://news.ycombinator.com"
+     public_addr: "hn.asteroid.earth
+```
+
+#### Start Teleport
+
+```
+$ teleport start -d --config=/etc/teleport.yaml
+```
+
+#### Update DNS
+
+In the above config example, I've a range of Applications with `public_addr`, these need to be
+setup with DNS using an `A, CNAME or AAAA` record point to the IP of the Teleport Proxy Server.
+
+For the beta, we would recommend using a wildcard cert for TLS. You can also use an individual certificate on the Teleport Main process using the new `https_keypairs` option.
+
+```yaml
+proxy_service:
+  #... Example Snippet for new https_keypairs.
+  https_keypairs:
+  - key_file: /etc/letsencrypt/live/jwt.asteroid.earth/privkey.pem
+  - cert_file: /etc/letsencrypt/live/jwt.asteroid.earth/fullchain.pem
+  - key_file: /etc/letsencrypt/live/hn.asteroid.earth/privkey.pem
+  - cert_file: /etc/letsencrypt/live/hn.asteroid.earth/fullchain.pem
+  - key_file: /etc/certs/privkey.pem
+  - cert_file: /etc/certs/fullchain.pem
+```
+
+## View Applications in Teleport
+
+`https://[cluster-url]:3080/web/cluster/[cluster-name]/apps`
+
+<video autoplay loop muted playsinline style="width:100%">
+  <source src="/video/app.mp4" type="video/mp4">
+  <source src="/video/app.webm" type="video/webm">
+Your browser does not support the video tag.
+</video>
+
+## Product Feedback
+
+We really value early feedback for our products. [You can easily pick a time here to have a chat on your feedback.](https://calendly.com/c/AEBVTDEDCQ2Z2YSB?month=2020-10)
+
+### Found a bug?
+
+Please create a [Github Issue](https://github.com/gravitational/teleport/issues/new/choose)
```

Comment:
All the changes I suggested to the 4.4 file also need applying here.

Reply:
I've removed the Teleport Application Access from 5.0